pynamodb/models.pyi

```diff
@@ -30,6 +30,10 @@ class Model(metaclass=MetaModel):
     _range_keyname: Optional[str]
     _connection: Optional[TableConnection]
     def __init__(self, hash_key: Optional[KeyType] = ..., range_key: Optional[Any] = ..., **attrs) -> None: ...
+    @property
+    def _hash_key(self) -> KeyType: ...
+    @property
+    def _range_key(self) -> Optional[Any]: ...
```

Comment:
why not `Optional[KeyType]`? (I recognize that the `__init__` has `range_key: Optional[Any]`, but I'm curious why that it for both)